test/common/upstream/load_balancer_impl_test.cc

```diff
@@ -427,13 +446,32 @@ TEST_P(LoadBalancerBaseTest, GentleFailoverWithExtraLevels) {
 
   // Rounding errors should be picked up by the first priority with degraded hosts when
   // there are no healthy priorities.
+  // Just disable panic threshold. Otherwise TotalPanic calculation
```

Comment:
minor nit but I would rephrase this as `Disable panic threshold to prevent total panic from kicking in`. Alternatively, tweak the parameters of the test so that we're still able to test the behavior of rounding when everything is degraded without having to disable panic.

Reply:
Done.